lib/shopify_app/engine.rb

```diff
@@ -21,5 +31,16 @@ class Engine < Rails::Engine
         app.config.middleware.insert_after(ShopifyApp::SameSiteCookieMiddleware, ShopifyApp::JWTMiddleware)
       end
     end
+
+    initializer "shopify_app.redact_job_params" do
+      ActiveSupport.on_load(:active_job) do
+        if ActiveJob::Base.respond_to?(:log_arguments?, true)
```

Comment:
I had to look up what the boolean argument meant here. Perhaps we could add an explaining variable, e.g. `include_private_and_protected = true`?

Reply:
I think I can just take it out, log_arguments is a public attribute, I forget why I felt the need to check private scope as well.